guide/sections/part2/global-services.adoc

```diff
@@ -79,9 +79,10 @@ In the following sections and for each Global Service, a set of metrics is defin
 ===== Technical considerations
 
 * As detailed above, there will be at least three (3) instances of Global Broker to ensure highly available, low latency global provision of messages within WIS.
-* A Global Broker instance subscribes to messages from WIS Centres, Global Caches and other Global Brokers. The Global Broker should aim at subscribing to all WIS Centres. If this is not possible, for whatever reason, the Global Broker should inform WMO Secretariat so that situation is documented.
+* A Global Broker instance subscribes to messages from WIS Centres and other Global Services. The Global Broker should aim at subscribing to all WIS Centres. If this is not possible, for whatever reason, the Global Broker should inform WMO Secretariat so that situation is documented.
 * Every WIS Centre or Global Service must have subscriptions from at least two Global Brokers.
 * For full global coverage, a Global Broker instance will subscribe to messages from at least two (2) other Global Brokers.
+* When subscribing to messages from WIS Centres and other Global Services, a Global Broker must authenticate using the valid credentials provided by WMO Secretariat. 
```

Comment:
I'd prefer: 

* When subscribing to messages from WIS Centres and other Global Services, a Global Broker must authenticate using the valid credentials managed by the WIS Centre and available at WMO Secretariat.